### ISSUE TITLE
there is no response from alterts

### ISSUE THREAD

**Issue by timae146** [bondrogeen/ioBroker.paw#21]
![bla](https://user-images.githubusercontent.com/42387818/50015396-c5169880-ffc6-11e8-88c0-d1162ea49d40.PNG)

there is only a response error.

**Comment by bondrogeen** [comment]
> ![bla](https://user-images.githubusercontent.com/42387818/50015396-c5169880-ffc6-11e8-88c0-d1162ea49d40.PNG)
> 
> there is only a response error.


Or you do not have a dialog box?

**Comment by bondrogeen** [comment]
http://192.168.178.34:8080/control.html


Follow the links and check the work through the web interface

![image](https://user-images.githubusercontent.com/17860636/50138077-554a2b80-02ae-11e9-99ec-4056f2c28d06.png)

**Comment by timae146** [comment]
okay. works. ;)
but it seems to be a little screen bug there. response state only refresh if i refresh the Iobroker website (F5) but its could be a ioberoker/ browser bug too.

http://192.168.178.34:8080/control.html  isnt reachable. 
I´m forwarded to http://192.168.178.34:8080/index.html every time.

**Comment by bondrogeen** [comment]
> http://192.168.178.34:8080/control.html isnt reachable.
> I´m forwarded to http://192.168.178.34:8080/index.html every time.


Such a problem exists, and I do not know why it arises.
If the page redirects to index.html, press F12 and refresh the page again.
I still don’t know why this is happening.

**Comment by timae146** [comment]
> > http://192.168.178.34:8080/control.html isnt reachable.
> > I´m forwarded to http://192.168.178.34:8080/index.html every time.
> 
> Such a problem exists, and I do not know why it arises.
> If the page redirects to index.html, press F12 and refresh the page again.
> I still don’t know why this is happening.

mhm dont work.

but i noticed the ip adress of your test device in my settings. and its not changeable. could it be linked with the problem?

![desktop](https://user-images.githubusercontent.com/42387818/50174010-4d63a900-02f9-11e9-83b0-12b8f6861eec.PNG)

**Comment by bondrogeen** [comment]
> but i noticed the ip adress of your test device in my settings. and its not changeable. could it be linked with the problem?

No, that's not the point. I'm still working on the web interface.

You still can not go to the page control.html?

**Comment by timae146** [comment]
> You still can not go to the page control.html?

no

**Comment by bondrogeen** [comment]
> > You still can not go to the page control.html?
> 
> no

Update the application.
Fixed bugs.
